google/cloud/bigtable/examples/README.md

```diff
@@ -3,25 +3,37 @@
 This directory contains minimal examples to get started quickly, and are
 referenced from the Doxygen landing page as "Hello World".
 
-## Compiling the Examples
+## Table of Contents
+
+- [Before you begin](#before-you-begin)
+  - [Compiling the Examples](#compiling-the-examples)
+  - [Running the examples against the Cloud Bigtable Emulator](#running-the-examples-against-the-cloud-bigtable-emulator)
+  - [Running the examples against a production version of Cloud Bigtable](#running-the-examples-against-a-production-version-of-cloud-bigtable)
```

Comment:
You don't mean "production version of Cloud Bigtable", you probably mean "Cloud Bigtable service", because you can use a `DEVELOPMENT` instance or a `PRODUCTION` instance (they work similarly from the perspective of API calls in this example), so the meaning of "production" is ambiguous here.

We don't want to imply that these API calls don't work on a `DEVELOPMENT` instance (e.g., table creation works just as well, but one thing you can't do in a development instance/cluster is add replication, or change number of nodes, for example).

Reply:
fixed